src/main/java/racingcar/InputViewCarName.java

```diff
@@ -0,0 +1,18 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputViewCarName<T> extends InputView<T>{
```

Comment:
> input 관련해서는 추상 클래스를 만들고, 오버라이딩 방식으로 했습니다. 제네릭을 사용해서 T로 반환했는데요. 제네릭을 사용하는게 맞을지 그냥 Object를 리턴해주고, 호출하는쪽에서 (String[]) 이런식으로 캐스팅하는게 좋을지 궁금합니다.

잘 바꿔 주셨어요! `InputViewCarName` 는 구체 클래스이므로 타입을 지정해 줄 수 있을것 같아요. 그럼 캐스팅도 필요 없어져요.
```suggestion
public class InputViewCarName extends InputView<String[]>{
...
```